test/e2e/auth/cas-sign-in.e2e-spec.ts

```diff
@@ -12,24 +11,10 @@ import { HttpStatus } from '@nestjs/common';
 
 const CasSignInE2ESpec = e2eSuite('POST /auth/signin/cas', (app) => {
   const body: AuthCasSignInReqDto = {
-    service: cas.validService,
     ticket: cas.validTicket,
     tokenExpiresIn: cas.user.tokenExpiresIn,
   };
 
-  it('should fail as provided service is not valid', async () => {
-    await pactum
-      .spec()
-      .post('/auth/signin/cas')
-      .withBody({ ...body, service: 'wrong service' })
-      .expectAppError(ERROR_CODE.INVALID_CAS_TICKET);
```

Comment:
Il faudra d'ailleurs renomme la description de l'erreur "The ticket or the service is invalid" en quelque chose comme "le ticket n'est pas valide" (enfin en anglais)